src/components/SandboxBlock/SandboxBlock.tsx

```diff
@@ -0,0 +1,196 @@
+import {ChevronsExpandUpRight} from '@gravity-ui/icons';
+import {
+    Col,
+    Icon,
+    RadioButton,
+    Row,
+    Select,
+    Switch,
+    Text,
+    TextInput,
+    Theme,
+} from '@gravity-ui/uikit';
+import {FC, MutableRefObject, ReactNode, useEffect, useRef, useState} from 'react';
+
+import themeIcon from '../../assets/icons/theme.svg';
+import {block} from '../../utils';
+
+import './SandboxBlock.scss';
+import {configOptions} from './config';
+import type {OptionType, SandboxBlockTypes} from './types';
+
+const b = block('sandbox-block');
+
+const SandboxBlock: FC<SandboxBlockTypes> = ({component}) => {
+    const [props, setProps] = useState({});
+    const [isFullScreen, setIsFullScreen] = useState(false);
+    const [globalTheme, setTheme] = useState<Theme>('dark');
+    const iframeRef = useRef() as MutableRefObject<HTMLIFrameElement | null>;
+
+    useEffect(() => {
+        iframeRef.current?.contentWindow?.postMessage(
+            {
+                ...props,
+                globalTheme,
```

Comment:
Maybe it's better to separate these props? Otherwise `globalTheme` will be placed in the components props.
```
{
    pageProps: { theme: globalTheme },
    componentProps: props,
}
```